src/test/kotlin/io/github/kryszak/gwatlin/http/serializers/SerialNameDelegateTest.kt

```diff
@@ -0,0 +1,25 @@
+package io.github.kryszak.gwatlin.http.serializers
+
+import io.github.kryszak.gwatlin.api.gamemechanics.model.facts.BuffConversion
+import io.github.kryszak.gwatlin.api.guild.model.log.GuildLogInvited
+import io.github.kryszak.gwatlin.api.items.model.item.ArmorItem
+import io.github.kryszak.gwatlin.api.items.model.item.ItemRarity
+import io.github.kryszak.gwatlin.api.items.model.item.ItemType
+import io.kotest.core.spec.style.FunSpec
+import io.kotest.matchers.shouldBe
+
+class SerialNameDelegateTest : FunSpec({
```

Comment:
internal modifier is missing
+ 
extend ShouldSpec to keep consistent pattern in tests

Reply:
Fixed :+1: